packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/index.tsx

```diff
@@ -9,6 +9,7 @@ import { TrezorDevice, AcquiredDevice, InjectedModalApplicationProps } from '@su
 import { useSelector, useActions } from '@suite-hooks';
 import * as routerActions from '@suite-actions/routerActions';
 import * as suiteActions from '@suite-actions/suiteActions';
+import { OpenGuideFromTooltip } from '@suite/views/guide';
```

Comment:
```suggestion
import { OpenGuideFromTooltip } from '@suite-views/guide';
```
Edit everywhere please

Reply:
90b9ca6cb461e549a88b4b1006dbaa18ad07f287